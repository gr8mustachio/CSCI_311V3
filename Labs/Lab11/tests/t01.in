6 10
0 3
0 4
3 2
3 4
1 2
1 5
2 5
4 1
4 2
4 5
printNeighbors 4
printNeighbors 3
printNeighbors 2
printNeighbors 5
printNeighbors 1
