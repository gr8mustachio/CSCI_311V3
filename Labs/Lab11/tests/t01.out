Graph is:
0: 3 4 
1: 2 5 
2: 5 
3: 2 4 
4: 1 2 5 
5: 
Neighbors of 4 are: 
1 2 5 
Neighbors of 3 are: 
2 4 
Neighbors of 2 are: 
5 
Neighbors of 5 are: 

Neighbors of 1 are: 
2 5 
